extide/libs.gradle/nbproject/org-netbeans-modules-libs-gradle.sig

```diff
@@ -0,0 +1,167 @@
+#Signature file v4.1
```

Comment:
Just a result of moving to another cluster, I assume. CCing @lkishalmi

Reply:
Confirmed.